packages/core/helper-plugin/lib/src/components/GenericInput/tests/index.test.js

```diff
@@ -111,13 +111,15 @@ describe('GenericInput', () => {
       });
     });
 
-    test('does call onChange callback with null on empty value', () => {
+    test('does call onChange callback with undefined on empty value', () => {
       const spy = jest.fn();
       const { input } = setupNumber({ value: 1, onChange: spy });
 
       fireEvent.change(input, { target: { value: '' } });
 
-      expect(spy).toHaveBeenCalledWith({ target: { name: 'number', type: 'number', value: null } });
+      expect(spy).toHaveBeenCalledWith({
+        target: { name: 'number', type: 'number', value: undefined },
+      });
     });
```

Comment:
I think now that this doesn't test anything regarding the `GenericComponent` anymore, you could remove the whole test, because it would test some design-system internals and should be done there.

Reply:
nice catch thanks!